docs/optimization_guide/nncf/ptq/code/ptq_onnx.py

```diff
@@ -16,7 +16,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # onnx.ModelProto object
+import onnx
+model = onnx.ModelProto() # onnx.ModelProto object
```

Comment:
```suggestion
import onnx
model = onnx.load("model path")
```

I hope it is more readable.